### PR TITLE
added response of post request

### PR DESCRIPTION
**Pull request by Chibbluffy**
Not totally sure how to mock test this. What do you think?

### CODE REVIEW

**Comment by jmhossler** [comment]
I don't really see what this gives the user. In the CLI context, they wouldn't get the return value. I suppose if someone imported this and wrote their own custom thing (I don't necessarily see why?). You could try to make it so that the cli script receives the return and does some logging if it isn't 200 or something like that, but right now this doesn't seem to offer any real value.

File: tests/test_notify_me.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 class TestNotifyMeSuite:
     @patch('notify_me.script.requests')
     def test_send_message(self, requests_mock):
```
Comment (jmhossler) [inline tests/test_notify_me.py]: after this line, put requests.post.return_value == MagicMock(status_code=200)

File: tests/test_cli_script.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from click.testing import CliRunner
-from unittest.mock import patch
```
Comment (jmhossler) [inline tests/test_cli_script.py]: I'm confused; why did you need to make this change? This wasn't an issue in my environment.

Comment (Chibbluffy) [reply tests/test_cli_script.py]: So, before I ran 'python setup.py test' I was just running 'pytest' and that's what it wanted. I just forgot to change it back, but it works for both with 'python setup.py test'

Comment (Chibbluffy) [reply tests/test_cli_script.py]: would you like me to revert it?

Comment (jmhossler) [reply tests/test_cli_script.py]: yes pls

File: tests/test_notify_me.py [hunk 3]
```diff
@@ -1,11 +1,10 @@
-from unittest.mock import patch
```
Comment (jmhossler) [inline tests/test_notify_me.py:1]: can we keep it consistent

File: tests/test_notify_me.py [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
 from notify_me.notify_me import notify_me
 
-
 class TestNotifyMeSuite:
     @patch('notify_me.notify_me.requests')
     def test_send_message(self, requests_mock):
-        notify_me('message', {'url': '127.0.0.1'})
+        response = notify_me('message', {'url': '127.0.0.1'})
 
+        requests_mock.post.return_value == MagicMock(status_code=200)
```
Comment (jmhossler) [inline tests/test_notify_me.py:9]: also, call assert before this boolean statement. Otherwise it isn't verifying it.

**Comment by jmhossler** [comment]
this has a lot of faults with it and is generally complete.
